sim/pkg/serve/run_handler.go

```diff
@@ -346,6 +346,7 @@ func buildClusterConfig(srr *SkenarioRunRequest) model.ClusterConfig {
 		TerminateDelay:          srr.TerminateDelay,
 		NumberOfRequests:        uint(srr.UniformConfig.NumberOfRequests),
 		InitialNumberOfReplicas: srr.InitialNumberOfReplicas,
+		MetricsPipelineLag:      1 * time.Millisecond,
```

Comment:
Let's set a more realistic default, e.g. `10 * time.Second`.

Reply:
Changed it